pkg/controllers/binding/common.go

```diff
@@ -193,6 +197,31 @@ func RecordAppliedOverrides(cops *overridemanager.AppliedOverrides, ops *overrid
 	return annotations, nil
 }
 
+// mergeConflictResolution determine the conflictResolution annotation of Work: preferentially inherit from RT, then RB
+func mergeConflictResolution(workload *unstructured.Unstructured, conflictResolutionInBinding policyv1alpha1.ConflictResolution,
+	annotations map[string]string) map[string]string {
+	// conflictResolutionInRT refer to the annotation in ResourceTemplate
+	conflictResolutionInRT := util.GetAnnotationValue(workload.GetAnnotations(), workv1alpha2.ResourceConflictResolutionAnnotation)
+
+	// the final conflictResolution annotation value of Work inherit from RT preferentially
+	// so if conflictResolution annotation is defined in RT already, just copy the value and return
+	if conflictResolutionInRT == workv1alpha2.ResourceConflictResolutionOverwrite || conflictResolutionInRT == workv1alpha2.ResourceConflictResolutionAbort {
+		annotations[workv1alpha2.ResourceConflictResolutionAnnotation] = conflictResolutionInRT
+		return annotations
+	} else if conflictResolutionInRT != "" {
+		// ignore its value and add logs if conflictResolutionInRT is neither abort nor overwrite.
+		klog.Errorf("ignore the invalid resource conflict-resolution annotation in ResourceTemplate: %s", conflictResolutionInRT)
```

Comment:
How about `klog.Warningf`?

Reply:
Print out the resource information, at least namespace/name.